docs/roadmap-2021.md

```diff
@@ -2,6 +2,13 @@
 
 ## Next 3 Months (Short-term)
 
+### Knative Eventing 1.0
+Releasing Knative Eventing v1.0.
```

Comment:
I think the goal is "Make current feature set 1.0-ready" rather than releasing 1.0. The actual 1.0 release depends on other things outside our control.

Reply:
Done